api/src/functions/health-check/index.ts

```diff
@@ -0,0 +1,13 @@
+// environment glue
+module.exports = function (context: any, req: any): any {
+    const timeStamp = new Date().toISOString();
+    
+    context.log('Azure Health Check function is running at -- ', timeStamp);  
+    
+    context.res = {
+        status: 200,
+        body: `Health Check ran at ${timeStamp}`,
```

Comment:
Let's just have this return null. The API may eventually be used more broadly, and I don't want to set any kind of expectation for data/information returned by this endpoint. I think the logging is good, though.

Reply:
I believe that we could leave this as it is. Because if we are returning `null`, it is a bit harder to think that, okay I got a HTTP 200 so it must be fine. Other healthchecks which also work on load balancers do the same thing. It is pretty minimal. Let me know if you would still like to discuss this.